### ISSUE TITLE
Make Deep Scan an API parameter, so a library can be scanned deep only once

### ISSUE THREAD

**Issue by gotson** [gotson/komga#1137]
### Describe your suggested feature

A common issue with the deep scan is that people forgot to deactivate it after using it once. Deep scan is costly, and should only be used for troubleshooting, and not left on all the time.

Moving it from the Library option to an API parameter would allow to trigger a deep scan once.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.
